build.gradle

```diff
@@ -147,11 +147,36 @@ processResources {
   }
 }
 
+sourceSets {
+  inttest {
+    java.srcDir 'src/inttest/java'
+    resources.srcDir 'src/inttest/resources'
+    compileClasspath += main.output + test.output
+    runtimeClasspath += main.output + test.output
+  }
+}
+
+configurations {
+  inttestCompile.extendsFrom testCompile
+  inttestRuntime.extendsFrom testRuntime
+}
+
+
+task inttest(type: Test) {
+  group = LifecycleBasePlugin.VERIFICATION_GROUP
+  description = "Runs the integration tests."
+
+  testClassesDir = sourceSets.inttest.output.classesDir
+  classpath = sourceSets.inttest.runtimeClasspath
+  mustRunAfter test
+}
+
 // JACOCO & COVERALLS
 
 apply plugin: 'jacoco'
 apply plugin: 'com.github.kt3k.coveralls'
 jacocoTestReport {
+  additionalSourceDirs = files(sourceSets.inttest.allJava.srcDirs)
```

Comment:
This treats the `inttest` directory as application source code to be covered. That's why the coverage dropped 20%.

Reply:
No, the coverage dropped already before the last commit. This line was the bumbling try to raise it again.